src/test/resources/data/AllAccessControlUiTest.json

```diff
@@ -160,7 +160,7 @@
             "courseId":"idOfTypicalCourse1.access",
             "name":"Helper Course1",
             "email":"helper@course1.tmt",
-            "role":"Helper",
+            "role":"Custom",
             "isDisplayedToStudents":false,
             "displayedName":"Helper",
             "instructorPrivilegesAsText": "{\n  \"courseLevel\": {\n    \"canviewstudentinsection\": false,\n    \"cangivecommentinsection\": false,\n    \"cansubmitsessioninsection\": false,\n    \"canmodifysessioncommentinsection\": false,\n    \"canmodifycommentinsection\": false,\n    \"canmodifycourse\": false,\n    \"canviewsessioninsection\": false,\n    \"canmodifysession\": false,\n    \"canviewcommentinsection\": false,\n    \"canmodifystudent\": false,\n    \"canmodifyinstructor\": false\n  },\n  \"sectionLevel\": {},\n  \"sessionLevel\": {}\n}"
```

Comment:
Yes, we can wrap the lines similar to those in `FeedbackRankQuestionUiTest.json`.
```json
"instructorPrivilegesAsText": "{
    \"courseLevel\": {
        \"canviewstudentinsection\": true,
        \"cangivecommentinsection\": true,
        \"cansubmitsessioninsection\": true,
        \"canmodifysessioncommentinsection\": true,
        \"canmodifycommentinsection\": true,
        \"canmodifycourse\": true,
        \"canviewsessioninsection\": true,
        \"canmodifysession\": true,
        \"canviewcommentinsection\": true,
        \"canmodifystudent\": true,
        \"canmodifyinstructor\": true
    },
    \"sectionLevel\": {},
    \"sessionLevel\": {}
}"
```